### DISCUSSION TITLE
Tailwind Styles not rendering in the browser

### DISCUSSION THREAD

**Discussion by marktmcewan** [tailwindlabs/tailwindcss#12631] [Help]
Hi, Learning Tailwind and have worked my way through 4 full build tutorials successfully. Today I am on another new build tutorial and I am having issues with the Tailwind classes rendering in the browser. Everything has worked fine until today, I am even following the same notes from the previous tutorials but clearly i have made an error somewhere: screen recording below

*the classes that do render as expected have shown up in the style.css in the css folder, its like its not pulling in any alternative to those.

Any help/pointers would be appreciated. Thank you.

https://github.com/tailwindlabs/tailwindcss/assets/20793022/2d8bf691-396a-4695-84f0-c1283e45af7d

**Comment by crswll** [comment]
It's very hard to tell based on a video but it seems like you're not running Tailwind CSS in watch mode. For better help you should likely just share your repo.

**Reply by marktmcewan** [reply]
Hi Bill, I have followed the Tailwind CLI setup from the docs to include running `npx tailwindcss -i ./src/input.css -o ./build/css/style.css --watch` in the terminal. This has worked sucessfully on all the previous tutorials.
*I have not published to Github yet (if I get that far i will and share it here)

**Reply by Zehad-10** [reply]
Hi 

it's not working after reloading the page?

**Reply by marktmcewan** [reply]
Hi Zehad, It does not work on reload either.

**Reply by Zehad-10** [reply]
Can you please show us your `tailwind.config.js` file?

**Reply by marktmcewan** [reply]
`/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ["./build/**/*.{html,js}"],
  theme: {
    extend: {},
  },
  plugins: [],
}

`

**Reply by marktmcewan** [reply]
<img width="469" alt="Screenshot 2023-12-20 at 18 34 25" src="https://github.com/tailwindlabs/tailwindcss/assets/20793022/26bfb38b-eb1c-43bb-9fd8-b69b15749841">

Github is rendering the code different than it is. Screenshot attached.

**Reply by Zehad-10** [reply]
your `tailwind.config.js`  seems okay, did your terminal show any error?

**Reply by marktmcewan** [reply]
No error in the terminal. I have done the last few courses using my notes. Let me share them with you incase there is something obvious in there.

https://south-cry-10a.notion.site/TailwindCSS-TailwindUI-739aa87021364627bd84eae6930d7858?pvs=4

**Reply by Zehad-10** [reply]
Please check your terminal and see that your building command `rebuilding` your classes whenever you are saving the code

**Reply by marktmcewan** [reply]
Rebuilding works fine as i can change the H1 tag etc and when i CMD+S it instantly changes in the browser.
Am i correct in understanding that when i use a 'class' it is added to the style.css file ? AS these have been added but anything other then these as shown it does not render the change in the browser (its like tailwind has disconnected)
<img width="935" alt="Screenshot 2023-12-20 at 19 01 19" src="https://github.com/tailwindlabs/tailwindcss/assets/20793022/461a387a-e9f2-495c-8788-537d4eb3ae0c">

**Reply by crswll** [reply]
What do you have in `src`? It's strange your `content` array is `./build/**/*` and not `./src/**/*` unless you're building your project constantly and you want Tailwind to see those changes.
